Samples/RadarIO.Maui.Sample/MainPage.xaml.cs

```diff
@@ -0,0 +1,32 @@
+﻿using static RadarIO.Sample.Demo;
+
+namespace RadarIO.Maui.Sample;
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        Initialize(new RadarSDKImpl());
+    }
+
+    private async void OnCounterClicked(object sender, EventArgs e)
+    {
+        try
+        {
+            await Permissions.RequestAsync<Permissions.LocationAlways>();
+
```

Comment:
Is this the sample app sent via TestFlight?

If so, can you add three more buttons:
* startTrackingResponsive
* startTrackingContinuous
* stopTracking